src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/FeatureToggleService.java

```diff
@@ -1,30 +1,33 @@
 package uk.gov.hmcts.reform.finrem.caseorchestration.service;
 
+import com.google.common.collect.Maps;
 import lombok.Getter;
-import lombok.RequiredArgsConstructor;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.finrem.caseorchestration.model.Features;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.ApprovedOrder;
 
 import javax.validation.constraints.NotNull;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
+import static java.util.Arrays.asList;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.model.Features.APPROVED_CONSENT_ORDER_NOTIFICATION_LETTER;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.model.Features.ASSIGNED_TO_JUDGE_NOTIFICATION_LETTER;
 import static uk.gov.hmcts.reform.finrem.caseorchestration.model.Features.HWF_SUCCESSFUL_NOTIFICATION_LETTER;
 
 @Service
-@ConfigurationProperties(prefix = "feature-toggle")
+@ConfigurationProperties(prefix = "feature")
 @Configuration
 @Getter
 public class FeatureToggleService {
 
     @NotNull
-    private Map<String, String> toggle = new HashMap<>();
+    Map<String, String> toggle = new HashMap<>();
```

Comment:
Why shouldn't this be private?

Reply:
Test uses this directly to inject values.

Reply:
But if we use this as an actual bean (loaded by Spring) and we use the spring annotation to set up the properties values for each test class, we can make this private again, right?

Reply:
I've pushed a change regarding this.